dali/python/nvidia/dali/types.py

```diff
@@ -349,6 +351,7 @@ def ConstantNode(device, value, dtype, shape, layout, **kwargs):
         def _type_from_value_or_list(v):
             if not isinstance(v, (list, tuple)):
                 v = [v]
+                shape = []
```

Comment:
Nor used?

Reply:
Used in line 403. This was actually a necessary fix.

Reply:
Correct me if I'm wrong, but this is the scope of `_type_from_value_or_list` function and you are not changing the value of the `shape` variable from the outer scope.